UI/js/login.js

```diff
@@ -0,0 +1,19 @@
+
+$('#login').submit(function(evt) {
+    evt.preventDefault();
+    $.ajax({
+        type: 'POST',
+        url: '/login',
+        data: $('#login').serialize(),
+        success: function(data) {
+            window.location.href = '/'; // TODO: add a link to the correct path
+        },
+        error: function(data) {
+            var jsonResponse = data.responseJSON;
+            $('#invalid').html(`<div class="chip white-text red darken-4">${getErrorFromResponse(jsonResponse)}<i class="close material-icons">close</i></div>`);
```

Comment:
lets move `<div class="chip white-text red darken-4">XYZ<i class="close material-icons">close</i></div>` into a function in common.js incase we want to use it in other areas of the website

Reply:
This is quite specific to the login scree, I dont think it should be moved

Reply:
not really moved. I mean we can make a common component so others can use it later